docs/Reference/Hooks.md

```diff
@@ -867,6 +860,9 @@ channel.subscribe(function ({ fastify }) {
 })
 ```
 
+> **Note:** The TracingChannel class API is currently experimental and may undergo
+> breaking changes even in semver-patch releases of Node.js.
```

Comment:
I believe this is the main reason to keep the `dc-polyfill` installed, but maybe @tlhunter knows better here.